mitodl/micromasters
# Populate exam_run for ProctoredExamGrades

#### What are the relevant tickets?
Fix #3300

#### What's this PR do?
Populate exam_run for ProctoredExamGrades

#### How should this be manually tested?
Run the migration and make sure that `ProctoredExamGrade`s that had null for `exam_run` field, have that field now populated.
